currency_rate_update_oxr/models/res_company.py

```diff
@@ -10,3 +10,7 @@ class ResCompany(models.Model):
     openexchangerates_app_id = fields.Char(
         string='OpenExchangeRates.org App ID',
     )
+    openexchangerates_eod_rates = fields.Boolean(
```

Comment:
I would suggest adding this option to parent module as `use_last_end_of_day_rates` or similar, since in theory not only OXR could have such option (to reduce number of fields)